x/concentrated-liquidity/model/pool.go

```diff
@@ -226,10 +226,6 @@ func (p Pool) CalcActualAmounts(ctx sdk.Context, lowerTick, upperTick int64, liq
 		return sdk.Dec{}, sdk.Dec{}, types.ErrZeroLiquidity
 	}
 
-	if lowerTick >= upperTick {
```

Comment:
why is this case impossible? Its in the public API so could be called externally, no?

Reply:
`TicksToSqrtPrice` from the line below does the same check

Reply:
I see that makes sense